docs/Reference/Plugins.md

```diff
@@ -133,8 +133,7 @@ fastify.listen({ port: 3000 }, (err, address) => {
 ### async/await
 <a id="async-await"></a>
 
-*async/await* is supported by `after`, `ready`, and `listen`, as well as
-`fastify` being a [Thenable](https://promisesaplus.com/).
+*async/await* is supported by `after`, `ready`, and `listen`
```

Comment:
But it is a [thenable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise#thenables). All promises are thenables, but not all thenables are promises.